Kudu.Core/Infrastructure/DockerContainerRestartTrigger.cs

```diff
@@ -41,6 +46,8 @@ public static void RequestContainerRestart(IEnvironment environment, string reas
                     AppSubPath = appSubPath
                 };
 
+                KedaFunctionTriggerProvider.UpdateFunctionTriggerBindingExpression(functionTriggers, appSettings);
```

Comment:
can you instead update the `GetFunctionTriggers` methods to do this instead of doing it outside as an extra step? There is another code path that calls GetFunctionTriggers not from here and should also get this fix

Reply:
Thank you @ahmelsayed 

I introduce the change. It is what you are expected?